File: codewars/7kyu/maximum-multiple/dduck-maximum-multiple.py
# https://www.codewars.com/kata/maximum-multiple

max_multiple = lambda divisor, bound: (bound // divisor) * divisor;

print(max_multiple(2, 7), 6)
print(max_multiple(3, 10), 9)
print(max_multiple(7, 17), 14)
print(max_multiple(10, 50), 50)
print(max_multiple(37, 200), 185)
print(max_multiple(7, 100), 98)

File: codewars/7kyu/maximum-multiple/dduck-maximum-multiple.js
// https://www.codewars.com/kata/maximum-multiple

const maxMultiple = (divisor, bound) => Math.floor((bound / divisor)) * divisor;

console.log(maxMultiple(2, 7), 6);
console.log(maxMultiple(3, 10), 9);
console.log(maxMultiple(7, 17), 14);
console.log(maxMultiple(10, 50), 50);
console.log(maxMultiple(37, 200), 185);
console.log(maxMultiple(7, 100), 98);
